ui/apps/ui/src/app/components/collections-navigation/collections-navigation.component.ts

```diff
@@ -9,19 +9,21 @@ import { toNavigationLink } from './utils';
   template: `
     <div class="container--xxl navigation">
       <div id="sub-nav">
-        <a
-          class="nav-btn {{ link.label }}"
-          *ngFor="let link of navigationLinks"
-          [routerLink]="link.routerLink"
-          [routerLinkActiveOptions]="{
-            queryParams: 'ignored',
-            paths: 'exact',
-            matrixParams: 'ignored',
-            fragment: 'ignored'
-          }"
-          routerLinkActive="active"
-          [queryParams]="{ q: (q$ | async) }"
-          >{{ link.label }}</a
+        <ng-container *ngFor="let link of navigationLinks">
+          <a
+            *ngIf="link.label.toLowerCase() !== 'providers'"
```

Comment:
I don't like conditioning generic component based on the link's label name.

Reply:
done. I moved the exclusion of providers from the top menu to the nav config registry and applied the filter on the result